RFS/build.gradle

```diff
@@ -70,6 +71,23 @@ clean.doFirst {
     delete project.file("./docker/build")
 }
 
+// Utility task to allow copying required libraries into a 'dependencies' folder for security scanning
+tasks.register('copyDependencies', Copy) {
```

Comment:
Is this another good use case for Sync? It seems like we could run into the same issue with leaving old dependencies in here.

Reply:
It probably doesn't hurt to add just in case, but normally I'd expect a full clean each time by the Solutions pipeline that uses this.